### DISCUSSION TITLE
Is it a good idea to call Berty "Web 3"?

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [berty/berty#4594] [General]
Something about [this blog post](https://berty.tech/blog/building-web3/) gave me pause, and I think I finally figure out what it was:

Web 3 is that big marketing hullabaloo that Facebook and other big tech corporations seem to like a lot.
Do we really want to help push that big tech marketing effort forward?
Even if yes there is still a negative stigma surrounding it, almost screaming "I am a scam" even though this genuinely isn't.

Shouldn't this project instead be supporting things such as Aral Balkan's Small Web?
Which also seems to be a better description of what this project really is?

Be careful about who you associate this project with. It means the difference between a successful protocol and another silicon valley startup tying to sell vaporware to Elon Musk.

**Comment by cybertheory** [comment]
Yeahhh - I think its a good thing. Web3 mean decentralized web (P2P, Blockchain, Federation). Most projects are pretty scammy blockchain adjacent startups. Its good to educate people that there is more to web3.
